hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockInputStream.java

```diff
@@ -77,7 +81,10 @@ public class BlockInputStream extends BlockExtendedInputStream {
       new AtomicReference<>();
   private final boolean verifyChecksum;
   private XceiverClientFactory xceiverClientFactory;
-  private XceiverClientSpi xceiverClient;
+  private XceiverClientSpi xceiverClientGrpc;
+  private XceiverClientShortCircuit xceiverClientShortCircuit;
+  private final AtomicBoolean fallbackToGrpc = new AtomicBoolean(false);
+  private FileInputStream blockInputStream;
```

Comment:
IMO the variable name blockInputStream is confusing because it's not a BlockInputStream after all.
What about something like 'chunkFileInputStream'

Reply:
Rename it to blockFileInputStream.